### ISSUE TITLE
Add task to setup gRPC loadbalancing with Istio 

### ISSUE THREAD

**Issue by rcaballeromx** [istio/istio.io#3254]
Demonstrate how gRPC loadbalancing with Istio works where mesh-external clients connect via GCP external and internal LoadBalancers.

**Comment by sanjaypujare** [comment]
So this covers the [ingress](https://istio.io/docs/tasks/traffic-management/ingress/) scenario? Is there a corresponding [egress](https://istio.io/docs/tasks/traffic-management/egress/) scenario that needs to be covered as well?

**Comment by rcaballeromx** [comment]
@sanjaypujare Yes, this issue is about configuring gRPC focusing on the ingress:
client_grpc_app (on GCE-VM) --> LB --> Istio --> Service
client_grpc_app (external) --> LB --> Istio --> Service
What path would you want for the egress?

**Comment by sanjaypujare** [comment]
A service (in a service-mesh) accessing endpoints external to the mesh or even external to GCP. There are 2 points to consider: policy enforcement and load balancing. gRPC LB can be used for LB but what about policy enforcement? Or there is no play here for gRPC LB?

I am just wondering and don't have specific suggestions or solutions.

**Comment by rcaballeromx** [comment]
@sanjaypujare Once I have a proposed draft in the form of a PR, it'll be easier for us to discuss specifics. We might need a separate doc for policy enforcement but it is hard to tell at this stage.

**Comment by kish3007** [comment]
is the grpc loadbalancing is working currently? because in my case load balancing is not working correctly with grpc services.

**Comment by esnible** [comment]
@kish3007 I did an experiment and was able to see gRPC load balancing.  I had difficulties with mTLS and GRPC but if I did not attempt to force mTLS settings it worked.

To test, I constructed a Service and a 2 replica Deployment using `image: fortio/fortio:latest_release` with `command: ["/usr/bin/fortio", "server", "-loglevel", "verbose"]` so that the server would log received traffic.

I then started an Ubuntu pod and installed Fortio using `go get fortio.org/fortio`.  I used the Fortio command line `fortio grpcping -n 100 fortio-echo` and saw traffic go to both replicas.  The traffic went to both pods without a DestinationRule, and with DestinationRule for load balancer ROUND_ROBIN, RANDOM, and LEAST_CONN.

I didn't have an easy way to distinguish between the three load balancer strategies with this small test, but I did see each pod getting a similar amount of traffic.
